pkgs/development/embedded/openocd-rp2040/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, fetchgit
+, pkg-config
+, hidapi
+, libftdi1
+, libusb1
+, which
+, libtool
+, autoconf
+, automake
+, texinfo
+, git
+, libgpiod
+}:
+
+stdenv.mkDerivation {
+  pname = "openocd-rp2040";
+  version = "0.11.0";
+  src = fetchgit {
+    url = "https://github.com/raspberrypi/openocd";
+    rev = "8e3c38f78730ce878ff81448bc3f6c1e6bb06e13";
+    sha256 = "sha256-4pY4Kzad53mmnnJDLl2FBDuulUldIo1NcNYzZKSCZts=";
+  };
```

Comment:
```suggestion

  src = fetchFromGitHub {
    owner = "raspberrypi";
    repo = "openocd";
    rev = "8e3c38f78730ce878ff81448bc3f6c1e6bb06e13";
    hash = "sha256-r0UcJ5/a51NLHtrTIBmWHZJ/ucVMB+NoJoTMtBRI9xM=";
  };
```

Reply:
actually the hash changed again, seems like upstream does do some whonky stuff, they also don't use tags for releases :confused: maybe open a issue and ask them to do proper releases.

Reply:
I'm trying to get in touch with the maintainers. raspberrypi/openocd#83 Let's see if they know anything and if we can resolve this with them.